src/main/java/step2/LottoResult.java

```diff
@@ -0,0 +1,52 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoResult {
+
+    private final List<LottoRank> lottoRanks;
+
+    public LottoResult(List<LottoRank> lottoRanks) {
+        this.lottoRanks = lottoRanks;
+    }
+
+    public LottoResult(LottoRank... lottoRanks) {
+        List<LottoRank> ranks = new ArrayList<>();
+        for (LottoRank lottoRank : lottoRanks) {
+            ranks.add(lottoRank);
+        }
+        this.lottoRanks = ranks;
+    }
```

Comment:
```suggestion
    public LottoResult(LottoRank... lottoRanks) {
        this.lottoRanks = new ArrayList<>(Arrays.asList(lottoRanks));
    }
```

이렇게 바로 선언해도 될것 같아요 :)